.travis.yml

```diff
@@ -17,18 +17,21 @@ matrix:
       env:
         - OS=ubuntu-14.04
         - JOBQUEUE=sge
+    - python: "3.6"
```

Comment:
I don't think you need to add a build entry for now since the tests without an environment marker will be run in all the builds as you can see from this [log](https://travis-ci.org/dask/dask-jobqueue/jobs/367037400#L2974) (some functions in test_pbs.py are run even if `JOBQUEUE=sge`).

Reply:
You are right, I was honestly not aware of this. However, I like the idea to have a simple test which does not depend on docker or on starting toy clusters. This could help to identify basic problems, don't you think so?

Reply:
OK fine with me.